### ISSUE TITLE
click twice is needed to Join a project in Join button for unregistered users.  

### ISSUE THREAD

**Issue by atfornes** [Grasia/teem#372]

**Comment by atfornes** [comment]
SessionSvc.loginRequired should be used propertly to solve this issue.

**Comment by krshubham** [comment]
[`join.js:40`](https://github.com/Grasia/teem/blob/master/src/js/directives/join.js#L40)
What is joinModel?
Should it be joinModal?

**Comment by atfornes** [comment]
@krshubham, that is right, well seen!

**Comment by atd** [comment]
U sure?

https://github.com/Grasia/teem/search?utf8=%E2%9C%93&q=join-model&type=

**Comment by atfornes** [comment]
Thanks @atd 

I did a quick search for joinModel and only saw that one, I forgot about the difference in capitalization used in the templates `join-model` vs `joinModel`. It is great you detect it!

**Comment by atd** [comment]
:wink: :+1:

**Comment by krshubham** [comment]
Yes it should be joinModel only. I was also wondering because it just stopped working! I should've have done that search before asking! 👍

**Comment by krshubham** [comment]
@atfornes 
can we please haev some steps to reproduce the issue, I tried this 2-3 times and I was not able to reproduce the issue. Thanks!

**Comment by atfornes** [comment]
@krshubham, the steps are as follows, if you cannot reproduce, we can close the issue.

1. as not logged in user click in a join button of a project
2. fill the form with your email and join text and click join
3. click start participating
4. register as new user
5. after registering, the user should be logged in and added to the project.

**Comment by krshubham** [comment]
@atfornes 
It does not happen in my dev setup, however I can see it happen on the currently deployed app! This should not happen 🤔
